### PR TITLE
feat: Sentry integration

### PR DESCRIPTION
**Pull request by puehringer**
### Developer Checklist (Definition of Done)

**Issue**

- [x] All acceptance criteria from the issue are met
- [x] Tested in latest Chrome/Firefox

**UI/UX/Vis**

- [ ] Requires UI/UX/Vis review
  - [ ] Reviewer(s) are notified (_tag assignees_)
  - [ ] Review has occurred (_link to notes_)
  - [ ] Feedback is included in this PR
  - [ ] Reviewer(s) approve of concept and design

**Code**

- [x] Branch is up-to-date with the branch to be merged with, i.e., develop
- [x] Code is cleaned up and formatted
- [ ] Unit tests are written (frontend/backend if applicable)
- [ ] Integration tests are written (if applicable)

**PR**

- [x] Descriptive title for this pull request is provided (will be used for release notes later)
- [x] Reviewer and assignees are defined
- [x] Add type label (e.g., *bug*, *feature*) to this pull request
- [x] Add release label (e.g., `release: minor`) to this PR following [semver](https://semver.org/)
- [x] The PR is connected to the corresponding issue (via `Closes #...`)
- [x] [Summary of changes](#summary-of-changes) is written

### Summary of changes

- Integrates Sentry into the `VisynApp`. The DSN in set in the backend, sent via the client config to the frontend, and used there to initialize the Sentry instance. 
- Caveat: If an error occurs before the VisynAppProvider is initialized, Sentry won't be initialized and thus no errors will be reported. 

### Additional notes for the reviewer(s)
To use, simply set to your DSN. This will be shared with the application via client config and used to initialize the Sentry instance.

```
VISYN_CORE__SENTRY__DSN=...
```

-  
Thanks for creating this pull request 🤗

### CODE REVIEW

File: visyn_core/settings/model.py [hunk 1]
```diff
@@ -154,6 +154,17 @@ class TelemetrySettings(BaseModel):
...
 
+class SentrySettings(BaseModel):
+    dsn: str | None = None
+    """
+    Public DSN of the Sentry project.
+    """
+    proxy_to: str | None = None
+    """
+    Proxy Sentry envelopes to this URL. Used if an internal Sentry server is used, otherwise the original DSN is used.
+    """
```
Comment (thinkh) [inline visyn_core/settings/model.py:165]: I suggest to harmonize the variable name. Either name this `tunnel` or rename the `sentry_tunnel` in the client config to `sentry_proxy_to` to make it easier to find the related variables.

**Review by thinkh — COMMENTED** [review]
In general it looks good to me. I have a minor comment regarding the variable naming.

File: visyn_core/settings/model.py (hunk 1 above)
Comment (puehringer) [reply visyn_core/settings/model.py:165]: :heavy_check_mark:

**Review by thinkh — APPROVED** [review]
Thanks. Looks good to me.

**Pull request merged**